docs/SNMP_Alarms.md

```diff
@@ -41,3 +41,13 @@ obtain the brief description, detailed description, and severity for the alarm:
 ## Alarm Models
 
 The alarm models used by Clearwater are defined in [alarmdefinition.h](https://github.com/Metaswitch/cpp-common/blob/master/include/alarmdefinition.h).
+
+## Alarm Resiliency
+
+Alarms are reported in such a way that if a component receiving alarm
```

Comment:
Actually I think that this could all be expressed a bit more concisely - "Clearwater is designed so that if an EMS fails and recovers, it will learn about any alarms raised by Clearwater while it was down, rather than those alarms just being lost."